weave/trace_server/clickhouse_trace_server_batched.py

```diff
@@ -1349,6 +1342,8 @@ def feedback_create(self, req: tsi.FeedbackCreateReq) -> tsi.FeedbackCreateRes:
             req.payload["detoned"] = detoned
             req.payload["detoned_alias"] = emoji.demojize(detoned)
             res_payload = req.payload
+        else:
+            res_payload = req.payload
```

Comment:
I think this is a tiny bug, unless this was an intentional omission? cc @jamie-rasmussen

Reply:
I think it was intentional - in the case of emojis the server is doing this extra work computing the alias, detoned version, etc. In other cases there's not really a point in sending the payload back to the client, which already has a complete version which it included in the request, it would just waste bandwidth.

Reply:
just seems weird you can get a complete payload for emojis but nothing for all others. I get it though, changing this slightly